src/config/file.rs

```diff
@@ -73,6 +74,8 @@ pub struct RawPlugin {
     /// If configured, only installs this plugin if one of the given profiles is
     /// set in the SHELDON_PROFILE environment variable.
     pub profiles: Option<Vec<String>>,
+    /// Hooks executed during template evaluation.
+    pub hooks: BTreeMap<String, String>,
```

Comment:
Sorry I should probably have been clearer. I think in the _config_ it should be an `Option<BtreeMap>` and in the _lock_ file it should be just a `BTreeMap` otherwise you get that thing where it adds the key unnecessarily to the config file when using `sheldon edit`

Reply:
Please have another look, that should be solved by my latest commit.